modules/postfix.lua

```diff
@@ -0,0 +1,522 @@
+-- This Source Code Form is subject to the terms of the Mozilla Public
+-- License, v. 2.0. If a copy of the MPL was not distributed with this
+-- file, You can obtain one at http://mozilla.org/MPL/2.0/.
+
+-- Copyright 2015 Mathieu Parent <math.parent@gmail.com>
+
+local string = require 'string'
```

Comment:
As a matter of style, you should use always the same kind of quotes (single or double) in a program, unless the string itself has quotes; then you use the other quote, or escape those quotes with backslashes.